render-utils.js

```diff
@@ -0,0 +1,36 @@
+export function renderProduct(beans) {
+    const li = document.createElement('li');
+    const name = document.createElement('h3');
+    const type = document.createElement('p');
+    const image = document.createElement('img');
+    const price = document.createElement('p');
+    const button = document.createElement('button');
+    
+    li.append(name, type, image, price, button);
+    li.classList.add('beans');
+    
+    name.classList.add('bname');
+    name.textContent = beans.name;
+   
+    image.classList.add('bbf');
+    image.src = `../assets/${beans.image}`;
+   
+    
+    type.classList.add('beanography');
+    type.textContent = beans.type;
+    
+
+    
```

Comment:
in the interest of readability/flow, try to keep excess white space to a minimum, using single blank lines to separate stanzas in your code poetry